frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -319,6 +330,88 @@ export const billingLogic = kea<billingLogicType>([
                 router.actions.replace('/organization/billing')
             }
             actions.registerInstrumentationProps()
+
+            if (values.productSpecificAlert) {
+                actions.setBillingAlert(values.productSpecificAlert)
+                return
+            }
+
+            if (!values.billing) {
+                // || !preflight?.cloud) {
+                return
+            }
+
+            if (values.billing.free_trial_until && values.billing.free_trial_until.isAfter(dayjs())) {
+                const remainingDays = values.billing.free_trial_until.diff(dayjs(), 'days')
+                const remainingHours = values.billing.free_trial_until.diff(dayjs(), 'hours')
+
+                if (remainingHours > 72) {
+                    return
+                }
+
+                actions.setBillingAlert({
+                    status: 'info',
+                    title: `Your free trial will end in ${
+                        remainingHours < 24 ? pluralize(remainingHours, 'hour') : pluralize(remainingDays, 'day')
+                    }.`,
+                    message: `Setup billing now to ensure you don't lose access to premium features.`,
+                })
+                return
+            }
+
+            if (values.billing.deactivated) {
+                actions.setBillingAlert({
+                    status: 'error',
+                    title: 'Your organization has been temporarily suspended.',
+                    message: 'Please contact support to reactivate it.',
+                    contactSupport: true,
+                })
+                return
+            }
+
+            const productOverLimit = values.billing.products?.find((x: BillingProductV2Type) => {
+                return x.percentage_usage > 1 && x.usage_key
+            })
+
+            if (productOverLimit) {
+                actions.setBillingAlert({
+                    status: 'error',
+                    title: 'Usage limit exceeded',
+                    message: `You have exceeded the usage limit for ${productOverLimit.name}. Please 
+                        ${productOverLimit.subscribed ? 'increase your billing limit' : 'upgrade your plan'}
+                        or data loss may occur.`,
+                    dismissKey: 'usage-limit-exceeded',
+                })
+                return
+            }
+
+            // debugger
+            // const { resetDismissKey } = lemonBannerLogic({ dismissKey: 'usage-limit-exceeded' }).actions
+            // resetDismissKey()
+            const logic = lemonBannerLogic.findMounted()
+            console.log('what', logic)
```

Comment:
This always prints `null`.

Reply:
`findMounted` takes a props argument, which you need to pass in to get the keyed logic with the right `dismissKey`